rpm/kubelet.spec

```diff
@@ -20,9 +20,14 @@ Source5: https://storage.googleapis.com/kubernetes-release/network-plugins/cni-a
 
 BuildRequires: curl
 Requires: iptables >= 1.4.21
+Requires: kubernetes-cni >= 0.3.0.1
 Requires: socat
 Requires: util-linux
 Requires: ethtool
+Requires: iproute2
```

Comment:
Have this been tested? AFAIK there is not package named iproute2 on RHEL and CentOS 7. This dependency can not be meet.